### PR TITLE
Add check to catch delinquent eng/common changes

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Add check to catch delinquent eng/common changes

### CODE REVIEW

**Comment by chidozieononiwu** [comment]
Please help approve @mitchdenny @weshaggard

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -181,6 +181,7 @@ jobs:
       name: ${{ parameters.LinuxPool }}
 
     steps:
+      - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
       - pwsh: |
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml:185]: nit: it would be nice to have a blank line here.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**